.mergify.yml

```diff
@@ -158,13 +158,13 @@ pull_request_rules:
       - "check-success=lvm-pvc"
       - "check-success=multi-cluster-mirroring"
       - "check-success=rgw-multisite-testing"
-      - "check-success=TestCephSmokeSuite (v1.19.16)"
-      - "check-success=TestCephSmokeSuite (v1.25.0)"
-      - "check-success=TestCephHelmSuite (v1.19.16)"
-      - "check-success=TestCephHelmSuite (v1.25.0)"
-      - "check-success=TestCephMultiClusterDeploySuite (v1.25.0)"
-      - "check-success=TestCephUpgradeSuite (v1.19.16)"
-      - "check-success=TestCephUpgradeSuite (v1.25.0)"
+      - "check-success=TestCephSmokeSuite (v1.21.14)"
```

Comment:
Let's just keep these changes in master and not backport. We don't want to change the min version in the 1.10 branch. We can keep these changes for v1.11.

Reply:
Still need to revert